src/java.base/share/classes/java/lang/foreign/Arena.java

```diff
@@ -61,7 +61,7 @@
  * by the garbage collector. The scope of an automatic arena is an automatic scope. As
  * such, the regions of memory backing memory segments allocated with the automatic arena
  * are deallocated at some unspecified time <em>after</em> the automatic arena (and all
- * the segments allocated by it) becomes <a href="../../../java/lang/ref/package.html#reachability">unreachable</a>,
+ * the segments allocated by it) becomes <a href="../../../java/lang/ref/package-summary.html#reachability">unreachable</a>,
```

Comment:
```suggestion
 * the segments allocated by it) becomes {@linkplain java.lang.ref##reachability unreachable},
```
same for other occurrences.

Reply:
TIL